go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/metal3-io/baremetal-operator
 
-go 1.21
+go 1.22
```

Comment:
This change does not feel appropriate for a stable branch.

Reply:
It is forced by Gophercloud v2.0.0. Currently we're running beta+commits here, which I'm not exactly fond of either.

I had assumption you were fine if it is forced by dependencies?

Reply:
/hold
Until discussion is finished.

Reply:
OK, let's keep the old version even if its not stable release and favor not bumping the go directive.